packages/api/src/command/medical/patient/get-patient.ts

```diff
@@ -68,17 +69,24 @@ export const getPatientByDemo = async ({
     ) {
       return true;
     }
-    // If the IDs don't match, or none were provided, check the demo for a match
-    let demoMatch =
-      intersectionWith(splitName(patient.data.firstName), splitName(demo.firstName), isEqual)
-        .length > 0 &&
-      intersectionWith(splitName(patient.data.lastName), splitName(demo.lastName), isEqual).length >
-        0 &&
-      intersectionWith(patient.data.address, demo.address, isEqual).length > 0;
-    if (demoMatch && demo.contact && demo.contact.length > 0) {
-      demoMatch = intersectionWith(patient.data.contact, demo.contact, isEqual).length > 0;
+
+    // Basic Implementation with semi-arbitrary threshold
+    const jw_score = calculatePatientSimilarity(patient.data, demo);
+    if (jw_score >= 0.95) {
```

Comment:
Can we move this to a constant at the top of the file so we can easily change it w/o looking through the code?